releases/v2.2.0-alpha.20190211.md

```diff
@@ -0,0 +1,209 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.2.0-alpha.20190211.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.2.0-alpha.20190211.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v2.2.0-alpha.20190211
+~~~
+
+### Backward-incompatible changes
+
+- The [`CHANGEFEED`](../v2.2/create-changefeed.html) `experimental_avro` format is now backward- and forward-compatible with adjacent schemas for the same table. [#34095][#34095] {% comment %}doc{% endcomment %}
+
+### General changes
+
+- [Go 1.11.4](https://golang.org/dl/) is now the minimum required version necessary to build CockroachDB. [#33668][#33668] {% comment %}doc{% endcomment %}
+- Increased the maximum length of queries in crash reports, to make debugging easier. [#34479][#34479]
+
+### Enterprise edition changes
+
+- [`CHANGEFEED`](../v2.2/create-changefeed.html)s now experimentally support writing to cloud storage, for easy use with analytics databases. [#33647][#33647] [#34193][#34193]
+- The `CHANGEFEED` `envelope=row` option is now deprecated and will be removed in the Fall 2019 release. The default envelope for new changefeeds is now `wrapped`. [#34309][#34309] {% comment %}doc{% endcomment %}
+- `CHANGEFEED`s now operate on an end-to-end "push" model, reducing latency of row changes. Some workloads will also see fewer transaction restarts on tables being watched by `CHANGEFEED`s. [#34457][#34457] {% comment %}doc{% endcomment %}
+- Added support for standard HTTP proxy environment variables in HTTP and S3 storage. [#34067][#34067]
+- Added support for performing sufficiently old historical reads against the closest replicas rather than leaseholders as well as a new `experimental_follower_read_timestamp()` [built-in function](../v2.2/functions-and-operators.html), which can be used with [`AS OF SYSTEM TIME`](../v2.2/as-of-system-time.html) clauses to generate a timestamp that is likely to be safe for reads from a follower. [#33478][#33478] {% comment %}doc{% endcomment %}
+
+### SQL language changes
+
+- [`VALIDATE CONSTRAINT`](../v2.2/validate-constraint.html) is now compatible with the new `MATCH FULL` and `MATCH SIMPLE` foreign key semantics and is more performant. [#34365][#34365] {% comment %}doc{% endcomment %}
+- Table data is now validated against a newly added [`CHECK`](../v2.2/check.html) constraint asynchronously after the transaction commits. [#32504][#32504] {% comment %}doc{% endcomment %}
+- `NULL` values are now supported in int and text arrays in the driver protocol. [#33675][#33675]
+- Added binary varbit support over pgwire. [#34050][#34050]
```

Comment:
Change to: "CockroachDB now supports transmitting bit array values using the decimal encoding in the low-level client protocol."